eng/bomgenerator/inputDependencies.txt

```diff
@@ -0,0 +1,24 @@
+com.azure:azure-core:1.17.0;
```

Comment:
These are the input dependencies that were used to generate BOM for azure-core 1.17.

Reply:
What is the plan to maintain this file?

Reply:
This file is used when BOM can't be generated from the current dependency set and you want to manually create a happy BOM. I will be removing this file and the batch file argument in the next cycle (no automation will be depending on this file), once azure-core 1.19 releases. This PR just brings us to the 'true state of things'.